challenge-programs: Add FAQ for the English version (#71)
pingcap/community: challenge-Programs/performance-challenge.md

- When the Contribution Credits owned by the participant is over "600" (including "600"),  "Easy" issues will be not valid for redeeming credits. 
- TiKV is a big codebase that may require some time to get familiar with. Easy issues are a great way to start if you plan larger contributions in the future. If you have significant upstream experience (e.g. you wrote a gRPC library), talk to us and we can bypass this requirement.

- Each individual participant or team can only pick up one issue at a time. Folks working on larger issues might stumble into fixes for smaller, Easy issues. That’s OK, submit these! And, don’t overburden yourself!

### Step 3: Fix the issue
If you have any questions during the coding process, you can discuss with us through **#performance-challenge-program** channel, and the mentor assigned for the issues will reply to you within 24h.

### Step 4: Commit your code
When your solution is ready, commit the code and submit a PR and to the corresponding GitHub Repo (i.e., tidb). Remember to request the review from the mentor responsible for this issue. The mentor will check your PR as soon as possible.

> How to submit:  see [GitHub Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md). There is also [SRE-BOT Command](https://github.com/pingcap/community/blob/master/contributors/command-help.md) Help for your reference.

Reviewer: * Seems like there is an extra space between the words "submit" and "see"
* The text of the link (https://github.com/pingcap/community/blob/master/contributors/command-help.md) is "SRE-BOT Command Help"

@BellaXiang @dcalvin PTAL

**Good luck and look forward to your brilliant idea!!**

## FAQ (Updating)

### Q1: How to change the registration information? How about the change of the Contribution Credit after the registration information changes?
If you want to change the signup information, you need to close the wrong signup Issue and open a new one for signing up. Editing on the original Issue is unavailable for the moment. There are three kinds of changes related to the Contribution Credit after the change of registration information:

- The total Contribution Credit owned by a team will not be affected by the change of team members;
- If you join a team, your individual Contribution Credit will remain at the state before joining the team, and all the Contribution Credit you gained after joining the team will be owned by the team;

> - If you quit a team, your original Contribution Credit is "0" because you can't inherit the team Contribution Credit.

Reviewer 2: What if the has credits before joining the team and now he quits to be individual again? He should still have the original credit according to point 1. @zz-jason

Author: Yes, "your original Contribution Credit is '0' " will be deleted. Thank you, Calvin!

### Q2: How to give up/change the picked Issue?

- Give up a Issue: commenting "give-up-challenge" on the picked Issue
- Change a Issue: giving up the Issue first, then picking up a new Issue

### Q3: How does the previous contributions in the community combine with the Performance Challenge Program?

> Performance Challenge Program is a brand new community activity embracing TiDB & TiKV open source community. While the previous contributions in the community can't be converted into the Contribution Credit, the participants who are Active Contributors and the superior role in the commnity can directly pick up the "Medium" and "Hard" Issues. (See [here](https://pingcap.com/community-cn/organization/) for more details about the roles in the commnity )

Reviewer 2: ```suggestion
Performance Challenge Program is a brand new community activity that embraces TiDB & TiKV open source community. While the previous contributions in the community can't be converted into the Contribution Credit, the participants who are Active Contributors or above in the commnity can directly pick up the "Medium" or "Hard" Issues. (See [here](https://pingcap.com/community-cn/organization/) for more details about the roles in the commnity).
```
